fearless/Modules/Profile/ProfilePresenter.swift

```diff
@@ -101,3 +104,31 @@ extension ProfilePresenter: Localizable {
         }
     }
 }
+
+private extension ProfilePresenter {
+    func updateAccountViewModel() {
+        guard let wallet = selectedWallet else {
+            return
+        }
+        let locale = localizationManager?.selectedLocale ?? Locale.current
```

Comment:
this class conforms to protocol Localizable, so you can get locale just by accessing "selectedLocale" property